src/libraries/Microsoft.PowerFx.Json/Functions/JsonUntypedObject.cs

```diff
@@ -21,7 +21,7 @@ internal class JsonUntypedObject : IUntypedObject
 
         public JsonUntypedObject(JsonElement element)
         {
-            _element = element;
+            _element = element.ValueKind == JsonValueKind.Undefined ? element : element.Clone();
```

Comment:
>element.Clone() [](http://example.com/codeflow?start=80&length=15)

We should double check whether we call this here, or have specific callers do it. 

Here impacts all scenarios, and there might be scenarios that *don't* want to clone.

Reply:
The problem is that we receive a JsonElement which is part of a disposable JsonDocument
If we don't clone it, we might lose the data